### PR TITLE
Fix Sequelize primary key settings

### PR DESCRIPTION
**Pull request by jonatas**
# Description

The actual structure is a bit inconsistent as we have to edit the migration manually and it was not exposed here.

I also added a warning to expose steps that are not necessary for cloud service.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)

Fixes https://github.com/timescale/docs/issues/551

### CODE REVIEW

File: timescaledb/quick-start/node.md [hunk 1]
```diff
@@ -157,6 +157,11 @@ Connection has been established successfully.
 Now that we have a successful connection to the `defaultdb` database, we
 can build out our first database and model.
 
+<highlight type="warning">
+You can skip the steps 1 and 2 if you're going to use TimescaleDB cloud. The
```
Comment (ryanbooz) [inline timescaledb/quick-start/node.md:161]: Link to either the Cloud sign-up page or the Docs "cloud getting started" page. @Loquacity or @lukas-bernert have thoughts?

Comment (Loquacity) [reply timescaledb/quick-start/node.md:161]: Point to the new cloud install page: timescaledb/:currentVersion:/how-to-guides/install-timescaledb/installation-cloud/

After https://github.com/timescale/docs/pull/583/ merges, that page will go live, note that we will be moving it again shortly, though.

Comment (Loquacity) [inline timescaledb/quick-start/node.md:161]:
```suggestion
You can skip the first two steps if you're going to use TimescaleDB cloud. The
```

File: timescaledb/quick-start/node.md [hunk 2]
```diff
@@ -157,6 +157,11 @@ Connection has been established successfully.
...
+service will create a database with the extension already enabled.
```
Comment (Loquacity) [inline timescaledb/quick-start/node.md:162]:
```suggestion
service creates a database with the extension already enabled.
```

**Review by lukas-bernert — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**